File: web-example/src/api/resp/PageInfo.ts

/**
 **/

export interface  PageInfo<T> {

    /**
     *响应集合列表
     *在java中的类型为：List
     **/
    records?: Array<T>;
    /**
     *查询页码
     *在java中的类型为：Integer
     **/
    queryPage?: number;
    /**
     *查询大小
     *在java中的类型为：Integer
     **/
    querySize?: number;
}

File: examples/swagger-3/src/test/resources/asserts/java/spring_http/src/com/wuxp/codegen/swagger3/resp/PageInfo.java
package com.wuxp.codegen.swagger3.resp;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import java.util.List;

@Data
@Accessors(chain = true)
public class  PageInfo<T> {

            /**
                *字段在java中的类型为：List
                *字段在java中的类型为：Object
            **/
        private List<T> records;

            /**
                *字段在java中的类型为：Integer
            **/
        private Integer queryPage;

            /**
                *字段在java中的类型为：Integer
            **/
        private Integer querySize;

}
